airbyte-cdk/python/airbyte_cdk/sources/declarative/extractors/http_selector.py

```diff
@@ -24,13 +24,15 @@ def select_records(
         stream_state: StreamState,
         stream_slice: Optional[StreamSlice] = None,
         next_page_token: Optional[Mapping[str, Any]] = None,
+        records_schema: Optional[Mapping[str, Any]] = None,
```

Comment:
why is this optional? I think this should always be set since the SchemaLoader always return a mapping

Reply:
fixed